### PR TITLE
Fix error: COPY failed: forbidden path outside the build context docker compose

### PR DESCRIPTION
**Pull request by s-a-tanjim**

### CODE REVIEW

File: .github/workflows/python-app-ci.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
         os: [ macos-latest, ubuntu-latest ]
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```
Comment (pfeairheller) [inline .github/workflows/python-app-ci.yml:23]: Why are we changing this version?

File: images/keripy.dockerfile [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+RUN apk add alpine-sdk
+RUN apk add libffi-dev
+RUN apk add libsodium
+RUN apk add libsodium-dev
 
 # Setup Rust for blake3 dependency build
 RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
 
-COPY .. /keripy
+COPY . /keripy
```
Comment (pfeairheller) [inline images/keripy.dockerfile:16]: This doesn't work from the Makefile.  Either revert this or fix the Makefile too.

**Review by pfeairheller — CHANGES_REQUESTED** [review]

File: .github/workflows/python-app-ci.yml (hunk 1 above)
Comment (s-a-tanjim) [reply .github/workflows/python-app-ci.yml:23]: Upgrading runtime `node12 --> node16`. It won't break anything, just making sure everything is up-to-date while developing.

File: images/keripy.dockerfile [hunk 3]
```diff
@@ -1,15 +1,19 @@
 
-FROM python:3.10.4-buster
+FROM python:3.10.4-alpine3.16
```
[the remark this answers is not in the document]
Comment (s-a-tanjim) [reply images/keripy.dockerfile:2]: Yes we can. I also ran the test, and it doesn't have any issue with the image.

**Review by pfeairheller — APPROVED** [review]

**Pull request merged**